ansible/roles/vm_set/tasks/renumber_topo.yml

```diff
@@ -86,6 +86,10 @@
     command: docker exec -i ptf_{{ vm_set_name }} sysctl -w net.ipv6.conf.all.disable_ipv6=0
     become: yes
 
+  - name: Set ipv6 route max size of ptf_{{ vm_set_name }}
```

Comment:
this only works after calling `restart-ptf`, add this task to `add-topo.yml` as well?

Reply:
Yes, you are right, should add this logic to add-topo as well.

Reply:
What's the value for IPv4?

Reply:
Very large 2147483647.